content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -300,6 +300,45 @@ does not support the `endPort` field and you specify a NetworkPolicy with that,
 the policy will be applied only for the single `port` field.
 {{< /note >}}
 
+## Targeting multiple namespaces by label
+
+In this scenario, your `Egress` NetworkPolicy targets more than one namespace using their
+label names. For this to work, you need to label the target namespaces. For example:
+
+```shell
+ kubectl label namespace frontend namespace=frontend
+ kubectl label namespace backend namespace=backend
+```
+
+Add the labels under `namespaceSelector` in your NetworkPolicy document. For example:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: egress-namespaces
+spec:
+  podSelector:
+    matchLabels:
+      app: myapp
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - namespaceSelector:
+        matchLabels:
```

Comment:
I'd suggest we use `matchExpressions` here for clarity. E.g.

```yaml
matchExpressions:
  - key: namespace
    operator: In
    values: ["frontend", "backend"]
```

The effect is the same but the readability is a little bit different.
You can provide multiple `NetworkPolicyIngressRule` in the `to` list, however, readers must be aware that these items are logical OR'ed during evaluation.

Reply:
Done. Thanks for the suggestion.